source/exe/posix/platform_main.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "common/common/macros.h"
+#include "common/common/thread_impl.h"
+
+#include "absl/debugging/symbolize.h"
+
+namespace Envoy {
+
+class PlatformMain {
+public:
+  PlatformMain(int, const char* const* argv) {
+#ifndef __APPLE__
+    // absl::Symbolize mostly works without this, but this improves corner case
+    // handling, such as running in a chroot jail.
+    absl::InitializeSymbolizer(argv[0]);
```

Comment:
should we have posix and windows implementions inherit from a common base-class that calls the symbolizer?

Reply:
...alternatively, you could just leave the call to InitializeSymbolizer in main() where it is now, and then you don't need to pass in argc,argv to PlatformMain.